index.bs

```diff
@@ -55,13 +55,46 @@ because, instead of waiting for a new kind of device to be popular enough for
 browsers to provide a specific API, new and innovative hardware can be built
 for the web from day one.
 
+# Motivating Applications # {#motivating-applications}
```

Comment:
Is the 3-D printer example from the explainer no longer a motivating application?